wallets/blink/client.js

```diff
@@ -1,10 +1,22 @@
-import { galoyBlinkUrl } from 'wallets/blink'
+import { getScopes, SCOPE_READ, SCOPE_RECEIVE, SCOPE_WRITE, getWallet, request } from 'wallets/blink/common'
 export * from 'wallets/blink'
 
 export async function testSendPayment ({ apiKey, currency }, { logger }) {
-  currency = currency ? currency.toUpperCase() : 'BTC'
   logger.info('trying to fetch ' + currency + ' wallet')
+  const scopes = await getScopes(apiKey)
+  if (!scopes.includes(SCOPE_READ)) {
+    throw new Error('missing READ scope')
+  }
+  if (!scopes.includes(SCOPE_WRITE)) {
+    throw new Error('missing WRITE scope')
+  }
+  if (scopes.includes(SCOPE_RECEIVE)) {
+    throw new Error('RECEIVE scope must not be present')
+  }
```

Comment:
I would say this check is not needed for sending, we only need to check that we can pay invoices. Stackers already struggle with attaching wallets, we don't need to make it harder if not necessary.